docs/getting-started/install-endpoint.asciidoc

```diff
@@ -65,17 +65,23 @@ If you have upgraded to an {stack} version that includes {fleet-server} 7.13.0 o
 [role="screenshot"]
 image::images/install-endpoint/add-agent.png[Add agent flyout on the Fleet page.]
 
-. Select an agent policy for the {agent}. You can select an existing custom policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+. Select an agent policy for the {agent}. You can select an existing policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
 +
-The selected policy should include {endpoint-sec}.
+The selected agent policy should include {endpoint-sec}.
 +
 [role="screenshot"]
 image::images/install-endpoint/endpoint-configuration.png[Add agent flyout with Endpoint Security integration highlighted.,575]
 
-. Download and extract the {agent} package on your host's machine.
+. Ensure that the **Enroll in {fleet}** option is selected. {endpoint-sec} cannot be integrated with an {agent} in standalone mode.
 
-. Open a command-line interface and navigate to the directory where the {agent} is located. Copy and run the command from {fleet} for your OS to enroll and run the agent.
+. Select the appropriate platform or operating system for the host, then copy the provided commands.
 
-. After you have enrolled the {agent} on your host, select **Close**. The host now appears on the **Endpoints** page in the {security-app}. It may take a minute or two while the {agent} starts up.
+. On the host, open a command-line interface and navigate to the directory where you want to install the {agent}. Paste and run the commands from {fleet} to download, extract, enroll, and start the {agent}.
+
+. (Optional) Return to the **Add agent** flyout in {fleet}, and observe the **Confirm agent enrollment** and **Confirm incoming data** steps automatically checking the host connection. It may take a few minutes for data to arrive in {es}.
+
+. After you have enrolled the {agent} on your host, you can click **View enrolled agents** to access the list of agents enrolled in {fleet}. Otherwise, select **Close**. 
```

Comment:
```suggestion
. After you have enrolled the {agent} instance on your host, you can click **View enrolled agents** to access the list of agents enrolled in {fleet}. Otherwise, select **Close**. 
```
I'm not sure if this is right but it seems like since this is specifically referring to a copy of agent on a particular machine, perhaps an article is appropriate? What do you think @jmikell821?

Reply:
I don't think you can install more than one instance of Elastic Agent on a machine/host, so maybe not "instance," but dropping the article here would be OK:
```suggestion
. After you have enrolled {agent} on your host, you can click **View enrolled agents** to access the list of agents enrolled in {fleet}. Otherwise, select **Close**. 
```

Reply:
Quick note, if we're dropping the article before Elastic Agent, there are a few other places that will need to be updated in this topic. That's outside the scope of this doc update though, so maybe it's a post-release task?

Reply:
@nastasha-solomon I think it could wait for post-release, if ever. I see this as a "fix it when you happen to notice it" kind of issue. Also, I didn't get the sense from @jmikell821's comment that we absolutely can no longer use "Elastic Agent' with an article, just that Fleet docs "often" omit the article.